src/app/routes/api/v3/admin/forms/__tests__/admin-forms.feedback.routes.spec.ts

```diff
@@ -0,0 +1,508 @@
+/* eslint-disable @typescript-eslint/ban-ts-comment */
+import { ObjectId } from 'bson-ext'
+import mongoose from 'mongoose'
+import supertest, { Session } from 'supertest-session'
+
+import { getEncryptedFormModel } from 'src/app/models/form.server.model'
+import getFormFeedbackModel from 'src/app/models/form_feedback.server.model'
+import getUserModel from 'src/app/models/user.server.model'
+import { IFormDocument, IUserSchema, ResponseMode, Status } from 'src/types'
+
+import {
+  createAuthedSession,
+  logoutSession,
+} from 'tests/integration/helpers/express-auth'
+import { setupApp } from 'tests/integration/helpers/express-setup'
+import dbHandler from 'tests/unit/backend/helpers/jest-db'
+import { jsonParseStringify } from 'tests/unit/backend/helpers/serialize-data'
+
+import { insertFormFeedback } from '../../../../../../modules/form/public-form/public-form.service'
+import { AdminFormsFeedbackRouter } from '../admin-forms.feedback.routes'
+
+// Prevent rate limiting.
+jest.mock('src/app/utils/limit-rate')
+
+const UserModel = getUserModel(mongoose)
+const EncryptFormModel = getEncryptedFormModel(mongoose)
+const FormFeedbackModel = getFormFeedbackModel(mongoose)
+
+const app = setupApp(undefined, AdminFormsFeedbackRouter, {
```

Comment:
can we have this router be set up with its real root route like `/admin/forms`?

Reply:
sure, will do